zephyr/CMakeLists.txt

```diff
@@ -210,6 +210,10 @@ if (CONFIG_SOC_SERIES_INTEL_ADSP_ACE)
 		${SOF_SRC_PATH}/schedule/zephyr_dp_schedule.c
 	)
 
+	zephyr_library_sources_ifdef(CONFIG_ZEPHYR_TWB_SCHEDULER
+		${SOF_SRC_PATH}/schedule/zephyr_twb_schedule.c
+	)
```

Comment:
Move this down to `src/schedule/CMakeLists.txt`, see recent example in 0419b7c5e98df2